cdn-index.template

```diff
@@ -0,0 +1,8 @@
+<html>
+<head><link rel="stylesheet" href="https://static.buildit.digital/gravity/v1.0.1/gravity.css"></head>
+<body>
+<h2>Versions</h2>
```

Comment:
Could you make this an `h1` please? It's the top level heading of this page, so we should use the appropriate semantics. :-)